core/src/main/java/org/apache/carbondata/core/datastore/page/SafeFixLengthColumnPage.java

```diff
@@ -289,6 +298,15 @@ public BigDecimal getDecimal(int rowId) {
     return data;
   }
 
+  @Override
+  public ByteBuffer[] getByteBufferArrayPage(boolean isFlattened) {
```

Comment:
The changes was only for offheap data right ? so I expect only unsafe pages should have changes. Why changed for safe column pages also ?